CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ Documentation for hipFFT is available at
 
 * `hipfftXtSetGPUs`, `hipfftXtMalloc, hipfftXtMemcpy`, `hipfftXtFree`, and `hipfftXtExecDescriptor` APIs
   have been implemented to allow FFT computing on multiple devices in a single process
+* CMake support for documentation.
```

Comment:
Can you merge current develop into this branch, and then make this an Addition under hipFFT 1.0.17?  i.e.
```md
## hipFFT 1.0.17 (unreleased)

### Additions
* CMake support for documentation.
```